src/conductor/client/workflow/conductor_workflow.py

```diff
@@ -195,7 +193,7 @@ def __get_workflow_task_list(self) -> List[WorkflowTask]:
         return workflow_task_list
 
     # Append task with the right shift operator `>>`
-    def __rshift__(self, task: TaskInterface | List[TaskInterface] | List[List[TaskInterface]]) -> Self:
+    def __rshift__(self, task: Union[TaskInterface, List[TaskInterface], List[List[TaskInterface]]]) -> Self:
```

Comment:
pytest couldn't collect these tests because the type hint is wrong here.

Reply:
Thanks for fixing this.